client/app/services/service-details/service-details.component.js

```diff
@@ -30,10 +30,10 @@ function ComponentController($stateParams, $state, $window, CollectionsApi, Even
       availableTags: [],
       credential: {},
       listActions: [],
-
       // Functions
       hasCustomButtons: hasCustomButtons,
       disableStopButton: disableStopButton,
+      displayedContent: 'details',
```

Comment:
I took a look at how catalogs had implemented something similar and I renamed the variable and implemented the same pattern.